coverage_comment/settings.py

```diff
@@ -66,6 +66,7 @@ class Config:
     ANNOTATION_TYPE: str = "warning"
     MAX_FILES_IN_COMMENT: int = 25
     USE_GH_PAGES_HTML_URL: bool = False
+    ACTIVITY: str | None = None
```

Comment:
You may define this as a `Literal["process_pr", "post_comment", "save_coverage_data_files"]`, this will give more precise typing. Also, you can call the validate at this step. I'd rather the arguments are checked as early as possible, so that the "business" code is straightforward.

(see https://typing.python.org/en/latest/spec/narrowing.html#typeguard)